website/src/pages/graphql.tsx

```diff
@@ -1,29 +1,32 @@
 import { graphql } from 'gatsby'
 import * as _ from 'lodash'
 import * as React from 'react'
-import Helmet from 'react-helmet'
-import BlogHeadLinks from '../components/BlogHeadLinks'
 import BlogPosts from '../components/BlogPosts'
 import Layout from '../components/Layout'
+
 export default class GraphQLSummitList extends React.Component<any, any> {
     constructor(props: any) {
         super(props)
     }
 
     public render(): JSX.Element | null {
+        const metaProps = {
+            title: 'GraphQL Summit 2017 Liveblog',
+            description: 'Checkout the official GraphQL Summit 2017 Liveblog proudly hosted by Sourcegraph.'
```

Comment:
```suggestion
            description: 'Check out the official GraphQL Summit 2017 Liveblog proudly hosted by Sourcegraph.'
```

Reply:
Good catch, have fixed with a commit that updated 2 other instances.